scripts/fgi

```diff
@@ -0,0 +1,259 @@
+#!/bin/bash
+# Copyright (c) The Diem Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -e
+set -o pipefail
+
+TAG=""
+PR=""
+WORKSPACE=""
+ENV=""
+LOCAL_BUILD=""
+EXIT_CODE=0
+# Default timeout is 45 mins
+TIMEOUT_SECS=2700
+
+FORGE_K8S_CLUSTERS=(rustietest)
+K8S_CONTEXT_PATTERN='arn:aws:eks:us-west-2:853397791086:cluster/libra-CLUSTERNAME'
```

Comment:
Ugh, I didn't realise we were divulging our AWS account number in the public repo; this isn't good.